internal/controller/apiextensions/composite/composite.go

```diff
@@ -27,3 +27,13 @@ func patchTypesToXR() []v1.PatchType {
 func patchTypesFromXR() []v1.PatchType {
 	return []v1.PatchType{v1.PatchTypeFromCompositeFieldPath, v1.PatchTypeCombineFromComposite}
 }
+
+// Returns types of patches that are _from_ the environment to a composed resource.
```

Comment:
Should it contain `ToEnvironment` patch types? If yes, please amend the comment

Reply:
Changed comment.